doc/sharedtable.md

```diff
@@ -0,0 +1,61 @@
+# sharedtable #
+
+```lua
+local t = ipc.sharedtable([tbl])
```

Comment:
It should be `ipc.sharedtable([tbl [,move]])`, where `move` is an argument that specify that we should move stuff out of the original table to the new one. This is useful in case the table is very big and we don't want to have 2 copies of the full data in memory.